scripts/validate_docstrings.py

```diff
@@ -26,6 +26,7 @@
 import importlib
 import doctest
 import tempfile
+import ast
```

Comment:
Does isort like this location?

Reply:
I didn't know about this.

Well, the already existing imports aren't properly sorted either. I could apply isort and commit the file, but I don't know if reordering all those imports can have any impact somewhere on the code. Thus, I'll refrain from doing that unless instructed.